.github/workflows/lint_and_test_cpp.yaml

```diff
@@ -257,13 +257,17 @@ jobs:
           - dsn_runtime_tests
           - dsn_utils_tests
           - dsn.zookeeper.tests
-          - partition_split_test
+          # TODO(yingchun): Disable it because we find it's too flacky, we will re-enable it after
+          # it has been optimized.
+#          - partition_split_test
```

Comment:
Actually `partition_split_test` and `dsn.meta.test` often failed for `test_Release` too, should both of them be disabled temporarily ?

Reply:
I've updated to disable `partition_split_test` in release type. But I think `dsn.meta.test` can be kept since it will not cost too much time if retring it.